packages/react-storage/package.json

```diff
@@ -71,7 +71,7 @@
       "name": "createStorageBrowser",
       "path": "dist/esm/browser.mjs",
       "import": "{ createStorageBrowser }",
-      "limit": "15.2 kB",
+      "limit": "16.7 kB",
```

Comment:
Not sure why this failed, guessing it was merging from main as I'm not adding any deps